libs/npm/core/src/utils/setCookie.ts

```diff
@@ -0,0 +1,41 @@
+export interface ICookieOptions {
+  path?: string;
+  domain?: string;
+  sameSite?: 'None' | 'Strict' | 'Lax';
+  secure?: boolean;
+  httpOnly?: boolean;
+  hostOnly?: boolean;
+  /** Date in GMT */
+  expires?: string;
+  /** Age in seconds */
+  maxAge?: number;
+}
+
+/**
+ * Adds the specified 'key' and 'value' to a cookie.
+ * @param key The cookie key name
+ * @param value The cookie key value
+ * @param options The cookie configuration options
+ */
+export const setCookie = (key: string, value: any, options?: ICookieOptions) => {
+  const config = {
+    sameSite: 'None',
+    secure: true,
+    expires: 'Fri, 31 Dec 9999 23:59:59 GMT',
+    ...options,
+  };
+  if (config.path && config.path.split('/').length > 1)
```

Comment:
It's possible to create multiple cookie values for deeper paths (i.e. `/admin/ingests`, `/admin/locations`).  However, cookies only support the first path value...  I find it hard to believe browsers still have this bug in them!